spec/rubocop/cop/layout/block_alignment_spec.rb

```diff
@@ -502,7 +537,7 @@ def get_gems_by_name
         def get_gems_by_name
           @gems ||= Hash[*get_latest_gems.map { |gem|
                            [gem.name, gem, gem.full_name, gem]
-                         }.flatten]
+          }.flatten]
```

Comment:
NOTE: It is corrected to follow the expected autocorrect behavior according to the documentation.